### PR TITLE
Update feature support statuses for different proposals

### PR DESCRIPTION
**Pull request by chicoxyzzy**

### CODE REVIEW

File: src/features/class-fields.md [hunk 1]
```diff
@@ -191,14 +191,14 @@ class Cat extends Animal {
 
 <feature-support chrome="72 /blog/v8-release-72#public-class-fields"
                  firefox="partial https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Releases/69#JavaScript"
-                 safari="no"
+                 safari="partial"
```
Comment (RReverser) [inline src/features/class-fields.md:194]: Are there any links and/or version numbers?

Comment (chicoxyzzy) [reply src/features/class-fields.md:194]: There are many bugzilla issues but only public class fields are available under --useClassFields JSC flag in Technology Preview for now.

Comment (RReverser) [reply src/features/class-fields.md:194]: Hmm so they don't have a meta-bug to track all of these? In that case let's link to one for public class fields where the flag is documented.

Comment (RReverser) [reply src/features/class-fields.md:194]: (same for other features / browsers in the PR)

**Pull request merged**